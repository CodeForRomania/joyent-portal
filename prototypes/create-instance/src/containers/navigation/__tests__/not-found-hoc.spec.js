/**
 * @jest-environment jsdom
 */

import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import { withNotFound } from '../';

it('renders <withNotFound /> without throwing', () => {
  const tree = renderer.create(<withNotFound />).toJSON();
  expect(tree).toMatchSnapshot();
});
