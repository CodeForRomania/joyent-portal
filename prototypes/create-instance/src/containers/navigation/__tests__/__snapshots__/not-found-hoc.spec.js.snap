// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <withNotFound /> without throwing 1`] = `<withNotFound />`;
